CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 ## Unreleased
 * Improved memory efficiency in **update-content-graph** and **create-content-graph** commands.
+* Added the `tenant_timezone` key to the test_modeling_rule file so that you can now control the timezone in the **modeling-rules test** command.
```

Comment:
Users (and frankly, I too) don't know what the `test_modeling_rule file` is. Write a changelog to make it clear to users of what changed and why.